### PR TITLE
Fixes #16286 - Move multiselect from gem to npm

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: webpack/assets/javascripts/bundle_multiselect.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+require('multiselect');
```
[the remark this answers is not in the document]
Comment (tbrisker) [reply webpack/assets/javascripts/bundle_multiselect.js:1]: looks like hound uses an older version of eslint that doesn't support this, what do you think about removing this rule @gailsteiger ?

[the remark this answers is not in the document]
Comment (gailsteiger) [reply webpack/assets/javascripts/bundle_multiselect.js:1]: Sure, as long as you're sure that hound is using the repository rules and not the default. We can pick it up in PR code review if anyone does this, which I hope is not likely.

[the remark this answers is not in the document]
Comment (tbrisker) [reply webpack/assets/javascripts/bundle_multiselect.js:1]: Done in #3787

**Comment by ehelms** [comment]
[test]

**Comment by dLobatog** [comment]
:+1: - please rebase @tbrisker and I'll merge, ping me if you want me to look at the PR in packaging

**Comment by tbrisker** [comment]
@dLobatog rebased, thanks

**Comment by tbrisker** [comment]
rebased again

**Comment by dLobatog** [comment]
Thanks - I'll take care of packaging of this one :smile:

**Comment by dLobatog** [comment]
Merged as 3f30919151ef899063afc14c71d791c55b1c96de, thanks @tbrisker!

**Pull request closed without merging**